src/components/Footer/__tests__/__snapshots__/Footer.test.tsx.snap

```diff
@@ -1,3 +1,3 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
```

Comment:
Had to re-gen this snapshot because copyright year in footer auto-updated from 2024 -> 2025